pkgs/servers/xmpp/prosody/default.nix

```diff
@@ -1,12 +1,14 @@
 { stdenv, fetchurl, lib, libidn, openssl, makeWrapper, fetchhg
 , lua
+, jitsi-meet-prosody
 , nixosTests
 , withLibevent ? true
 , withDBI ? true
 # use withExtraLibs to add additional dependencies of community modules
 , withExtraLibs ? [ ]
 , withExtraLuaPackages ? _: [ ]
 , withOnlyInstalledCommunityModules ? [ ]
+, withOwnerAllowKickPatch ? false
```

Comment:
Not sure if I like this approach. I think I would prefer something like `extraPatches ? []` and them we could pass `${jitsi-meet-prosody}/share/prosody-plugins/muc_owner_allow_kick.patch` as parameter when we want to apply the patch?

Not sure if it will work though, since you seem to be patching a specific file.